README.md

```diff
@@ -16,6 +16,10 @@ It includes Docker Compose configuration to run the application stack locally.
 
 <div style="clear: both;">&nbsp;</div>
 
+**🚧 NOTE**:
+Because this product was initially made as part of a hackathon under very tight time and resource limitations, its architecture had to be rapidly improvised, and as such is a bit "rough around the edges".
```

Comment:
Simplifying a bit. Also removing "rough..." (we can let them judge without our bias 🙂 ).
```suggestion
This product was initially made as part of a hackathon under very tight time and resource limitations which resulted in rapidly improvised architecture.
```